spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -240,6 +246,12 @@ def default_publishing_api_item(edition, override_hash = {})
       schema_name: edition.document_type.publishing_metadata.schema_name,
       details: {
         body: "",
+        change_history: [
+          {
+            note: "First published.",
+            public_timestamp: Time.zone.now,
+          },
+        ],
       },
     }.deep_merge!(override_hash)
```

Comment:
I feel like we could use a basic test in here that checks we are comparing change histories - It'd be fine if it mocks the change history check. I think we just enough that were we to remove https://github.com/alphagov/content-publisher/pull/1900/files#diff-93e7bf043d189b6ad9f28ef07d33176bR47-R52 the tests would fail.

Reply:
👍 thanks! This was useful in that I realised I needed to access the attributes of `proposed_change_history` and `publishing_api_change_history` with string keys rather than symbols in `ChangeHistoryCheck` due to the `PublishingApiPayload` being JSON.